doc/level1.rst

```diff
@@ -325,18 +323,18 @@ Updating Autocrypt Peer State
 Incoming messages may be processed to update Autocrypt peer state by a
 MUA at receive or display time.
 
-Messages SHOULD be ignored and the peer state SHOULD NOT be updated in
+Messages SHOULD be ignored, and the peer state SHOULD NOT be updated in
 the following cases:
 
-  - The content-type is ``multipart/report``. It can be assumed to be
-    auto-generated. This in particular avoids triggering a ``reset``
+  - The content-type is ``multipart/report``. In this case, it can be assumed
+    the message was auto-generated. This avoids triggering a ``reset``
     state from received Message Disposition Notifications (:rfc:`3798`).
 
   - There is more than one address in the ``From`` header.
 
   - The MUA believes the message to be spam. If the user marks the
```

Comment:
is the second sentence here even useful? seems to me we don't lose much by dropping it

Reply:
nack.  This changes the substance, I think.